XTS/raw-xts-api-demo/pom.xml

```diff
@@ -80,4 +64,21 @@
         <module>ear</module>
         <module>test</module>
     </modules>
+
+    <profiles>
+<!--     jdk17 profile is defined because org.jboss.arquillian.graphene.shaded.net.sf.cglib.core.ReflectUtils needs -->
+<!--     java.lang add-opens in order to work with JDK17  -->
+<!--     see https://docs.oracle.com/en/java/javase/16/migrate/migrating-jdk-8-later-jdk-releases.html#GUID-7BB28E4D-99B3-4078-BDC4-FC24180CE82B -->
+     <profile>
+          <id>jdk17</id>
+          <activation>
+            <jdk>[17,)</jdk>
+          </activation>
+          <properties>
+             <modular.jdk.args>
+              --add-opens=java.base/java.lang=ALL-UNNAMED
+            </modular.jdk.args>
+          </properties>
+      </profile>
+  </profiles>
```

Comment:
This profile is needed because this quickstart uses graphene that requires it. This is not visible from Jenkins because those tests are skipped there, but I found out it is required to be successful locally. This test is about graphene webdriver.